src/brs/http/GetAccount.java

```diff
@@ -18,6 +18,7 @@
 
   private final ParameterService parameterService;
   private final AccountService accountService;
+  private final String depreciationMessage = "For account assets use getAccountAssets. This will be removed after net hardfork";
```

Comment:
I think we should remove this message for now and include it in the future. We intend to promise no API changes for this next hard-fork. Maybe we can leave a TODO message on the code, but not active. There is also a typo on "net".